tracking/src/main/java/org/motechproject/nms/tracking/service/CacheService.java

```diff
@@ -0,0 +1,5 @@
+package org.motechproject.nms.tracking.service;
+
+public interface CacheService {
```

Comment:
ok, but this still feels like the wrong solution.  I think the correct fix would be to add cacheing to the platform on the entityService.getEntityByClassName method.  I don't see us doing that.

Is there an existing service in the tracking module that we could add that method to that would make sense?